pkg/tagger/tagstore.go

```diff
@@ -126,7 +145,21 @@ func (s *tagStore) prune() error {
 	s.storeMutex.Lock()
 	defer s.storeMutex.Unlock()
 	for entity := range s.toDelete {
-		delete(s.store, entity)
+		storedTags := s.store[entity]
```

Comment:
should we check if the entity exist in the `s.store`?  even if it should not be possible :)
I know it will create if not exist, but it could be a good short path condition.

Reply:
Good catch 👍 will do